### PR TITLE
Begin Mercury migration

### PR DESCRIPTION
**Pull request by wesley-tang**
Switched over from cheerio to mercury (esocket bugs exist, but do not seem to impact functionality)

### CODE REVIEW

File: lib/prepArticles.js [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
-			mainText.push("<p>" + $(this).html() + "</p>")
-		})
-
-		// Return all paragraph html joined by newlines
-		return callback(mainText.join(""))
+	Mercury.parse(url, { contentType: scrappedContentType }).then(parsedResult => {
+		return callback(parsedResult)
 	})
 	.catch(function(err){
+		console.log("Error has occured!!!11!!!\n\n")
```
Comment (brian-norman) [inline lib/prepArticles.js]: Pls remove

File: lib/prepArticles.js [hunk 2]
```diff
@@ -41,18 +37,18 @@ function prepAll(store, category, callback) {
...
-		            	result.push(newArticle)
-		            }
+					newArticle.text = parsedResult.content
+					newArticle.timeToRead = Math.ceil(parsedResult.word_count / AVG_WORDS_PER_MINUTE)
+					newArticle.sentiment = sentiment(parsedResult.content)
+					result.push(newArticle)
+		            
 
-		            if (completedArticles === articleList.length) {
+		            if (completedArticles >= articleList.length) {
```
Comment (brian-norman) [inline lib/prepArticles.js:50]: Reason?

**Review by brian-norman — CHANGES_REQUESTED** [review]
Looks good! If tested, then I like it woohoo 🎉 Pls remove one redundant print line

File: lib/prepArticles.js (hunk 2 above)
Comment (wesley-tang) [reply lib/prepArticles.js:50]: I thought I addressed this before lol, it's to account for glitches where article list length randomly exceeds the max

Comment (brian-norman) [reply lib/prepArticles.js:50]: Oh right on sounds good 👌

**Review by brian-norman — APPROVED** [review]

**Pull request merged**